<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Biblioteca - CRUD</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget {
    background-color: #f6f7fb;
    font-family: &quot;Segoe UI&quot;;
    font-size: 14px;
}

QLabel {
    color: #2c3e50;
}

QLabel#lblTitulo {
    font-size: 26px;
    font-weight: bold;
    color: #1f3a5f;
}

QLineEdit {
    background-color: #ffffff;
    color: #2c3e50;
    border: 1px solid #cfd3dc;
    border-radius: 6px;
    padding: 6px 8px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #4a90e2;
    background-color: #ffffff;
}

QDateEdit {
    background-color: #ffffff;
    color: #2c3e50;
    border: 1px solid #cfd3dc;
    border-radius: 6px;
    padding: 6px 8px;
    font-size: 14px;
}

QDateEdit:focus {
    border: 1px solid #4a90e2;
}

QDateEdit::up-button,
QDateEdit::down-button {
    width: 0px;
    height: 0px;
    border: none;
}

QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 14px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #3b7dc4;
}

QPushButton:pressed {
    background-color: #316aa3;
}

QTableWidget {
    background-color: #ffffff;
    border: 1px solid #cfd3dc;
    border-radius: 6px;
    gridline-color: #e5e7eb;
	color: rgb(0, 0, 0);
}

QHeaderView::section {
    background-color: #eef1f6;
    color: #2c3e50;
    font-weight: bold;
    padding: 6px;
    border: none;
}

QTableWidget::item {
    padding: 6px;
}

QTableWidget::item:selected {
    background-color: #dce9f9;
    color: #2c3e50;
}

QStackedWidget {
    background-color: transparent;
}

QScrollBar:vertical {
    width: 8px;
    background: transparent;
}

QScrollBar::handle:vertical {
    background: #cfd3dc;
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
    background: #b0b5bf;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="lblTitulo">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Sistema Biblioteca</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btnLibros">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Libros</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUsuarios">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Usuarios</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrestamos">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Préstamos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageUsuarios">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="labelUsuarios">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#labelUsuarios {
    background-color: #e9eef6;
    color: #1f3a5f;
    font-size: 18px;
    font-weight: bold;
    padding: 8px;
    border-radius: 6px;
    min-height: 40px;
    max-height: 40px;
    qproperty-alignment: AlignCenter;
}
</string>
          </property>
          <property name="text">
           <string>Gestión de Usuarios</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="txtId">
            <property name="placeholderText">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtNombre">
            <property name="placeholderText">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtCedula">
            <property name="placeholderText">
             <string>Cédula</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtCorreo">
            <property name="placeholderText">
             <string>Correo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTelefono">
            <property name="placeholderText">
             <string>Teléfono</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="btnCrearUsuario">
            <property name="text">
             <string>Crear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnListarUsuarios">
            <property name="text">
             <string>Listar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnModificarUsuario">
            <property name="text">
             <string>Modificar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEliminarUsuario">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBuscarUsuario">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePrestamos">
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="btnPrestamoRegistrar">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #ecf0f1;
    color: #2c3e50;
    border: 1px solid #cfd3dc;
}

QPushButton:hover {
    background-color: #e0e4e7;
}</string>
            </property>
            <property name="text">
             <string>Registrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPrestamoDevolver">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #ecf0f1;
    color: #2c3e50;
    border: 1px solid #cfd3dc;
}

QPushButton:hover {
    background-color: #e0e4e7;
}</string>
            </property>
            <property name="text">
             <string>Devolver</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPrestamoListar">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #ecf0f1;
    color: #2c3e50;
    border: 1px solid #cfd3dc;
}

QPushButton:hover {
    background-color: #e0e4e7;
}</string>
            </property>
            <property name="text">
             <string>Listar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPrestamoBuscar">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #ecf0f1;
    color: #2c3e50;
    border: 1px solid #cfd3dc;
}

QPushButton:hover {
    background-color: #e0e4e7;
}</string>
            </property>
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedPrestamosPages">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: #f0f0f0;
    color: #2c3e50;
    border: 1px solid #c0c0c0;
    border-radius: 6px;
    font-size: 15px;
    min-height: 30px;
    min-width: 100px;
}
</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pagePrestamoRegistrar">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLineEdit" name="txtPrestamoId">
              <property name="placeholderText">
               <string>ID Préstamo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtPrestamoLibro">
              <property name="placeholderText">
               <string>ID Libro</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtPrestamoUsuario">
              <property name="placeholderText">
               <string>ID Usuario</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="txtPrestamoFecha">
              <property name="dateTime">
               <datetime>
                <hour>5</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2026</year>
                <month>1</month>
                <day>26</day>
               </datetime>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnConfirmarPrestamo">
              <property name="text">
               <string>Confirmar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pagePrestamoDevolver">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLineEdit" name="txtDevolverPrestamoId">
              <property name="placeholderText">
               <string>ID Préstamo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="txtDevolverFecha">
              <property name="dateTime">
               <datetime>
                <hour>16</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2025</year>
                <month>1</month>
                <day>26</day>
               </datetime>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnConfirmarDevolucion">
              <property name="text">
               <string>Confirmar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pagePrestamoListar">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QTableWidget" name="tablePrestamos">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string>ID Préstamo</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ID Usuario</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ID Libro</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fecha Préstamo</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Estado</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRefrescarPrestamos">
              <property name="text">
               <string>Refrescar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageBuscarPrestamo">
           <widget class="QLineEdit" name="txtBuscarPrestamoId">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>631</width>
              <height>44</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>ID Usuario</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnBuscarPrestamos">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>210</y>
              <width>631</width>
              <height>35</height>
             </rect>
            </property>
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
           <widget class="QTableWidget" name="tablePrestamosBuscados">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>631</width>
              <height>121</height>
             </rect>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>ID Préstamo</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ID Usuario</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ID Libro</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fecha Préstamo</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Estado</string>
             </property>
            </column>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageLibros">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Page Libros</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
